R/brace_linter.R

```diff
@@ -8,9 +8,14 @@
 #'  - Closing curly braces in `if` conditions are on the same line as the corresponding `else`.
 #'  - Either both or neither branch in `if`/`else` use curly braces, i.e., either both branches use `{...}` or neither
 #'    does.
-#'  - Functions spanning multiple lines use curly braces.
+#'  - Function bodies are wrapped in curly braces.
 #'
-#' @param allow_single_line if `TRUE`, allow an open and closed curly pair on the same line.
+#' @param allow_single_line If `TRUE`, allow an open and closed curly pair on the same line.
+#' @param function_braces Character specifying whether to require function bodies to be wrapped in curly braces.
```

Comment:
```suggestion
#' @param function_braces Character string specifying whether to require function bodies to be wrapped in curly braces.
```

(NB: also need to `roxygenize()`)